apps/communications/dialer/style/call_log.css

```diff
@@ -163,7 +163,7 @@ ol, ul {
 }
 
 .log-item .primary-info {
-  text-overflow: ellipsis;
+  display: flex;
```

Comment:
Yikes, I'm surprised that we've never noticed this before it was indeed a problem. Thanks for this fix.